concepts/spsp.md

```diff
@@ -0,0 +1,13 @@
+## SPSP
+[SPSP](https://github.com/interledgerjs/ilp-protocol-spsp) (Simple Payment Setup Protocol) is an application layer protocol, which uses HTTPS endpoints to securely exchange 
+payment details, and then allows the user to send one or more Interledger packets. SPSP provides lightweight client-server 
```

Comment:
It uses the STREAM protocol so "one or more Interledger packets" seems a little confusing. Maybe instead it could be "allows the user to send value over Interledger"